tests/profiles/adobefonts_test.py

```diff
@@ -70,22 +88,33 @@ def test_check_find_empty_letters():
     """Validate that empty glyphs are found."""
     check = CheckTester(adobefonts_profile, "com.adobe.fonts/check/find_empty_letters")
 
-    # this font has inked glyphs for all letters
-    font = TEST_FILE("source-sans-pro/OTF/SourceSansPro-Regular.otf")
-    assert_PASS(check(font))
+    PASS_MSG = "No empty glyphs for letters found."
+
+    # this OT-CFF font has inked glyphs for all letters
+    ttFont = TTFont(TEST_FILE("source-sans-pro/OTF/SourceSansPro-Regular.otf"))
+    assert assert_PASS(check(ttFont)) == PASS_MSG
+
+    # this OT-CFF2 font has inked glyphs for all letters
+    ttFont = TTFont(TEST_FILE("source-sans-pro/VAR/SourceSansVariable-Italic.otf"))
+    assert assert_PASS(check(ttFont)) == PASS_MSG
+
+    # this TrueType font has inked glyphs for all letters
+    ttFont = TTFont(TEST_FILE("source-sans-pro/TTF/SourceSansPro-Bold.ttf"))
+    assert assert_PASS(check(ttFont)) == PASS_MSG
 
     # this font has empty glyphs for several letters,
     # the first of which is 'B' (U+0042)
-    font = TEST_FILE("familysans/FamilySans-Regular.ttf")
-    message = assert_results_contain(check(font), FAIL, "empty-letter")
-    assert message == "U+0042 should be visible, but its glyph ('B') is empty."
+    ttFont = TTFont(TEST_FILE("familysans/FamilySans-Regular.ttf"))
+    msg = assert_results_contain(check(ttFont), FAIL, "empty-letter")
+    assert msg == "U+0042 should be visible, but its glyph ('B') is empty."
 
 
 def _get_nameid_1_win_eng_record(name_table):
     """Helper method that returns the Windows nameID 1 US-English record"""
     for rec in name_table.names:
         if (rec.nameID, rec.platformID, rec.platEncID, rec.langID) == (1, 3, 1, 0x409):
             return rec
+    return None
```

Comment:
Not strictly needed, but fine.

Reply:
I know. Just wanted to not hear from pylint about it.